### ISSUE TITLE
Check Api seems to be broken with 1.9

### ISSUE THREAD

**Issue by mrudulp** [microsoft/playwright-python#560]
Here is a snippet which used to work with 1.8.0a
```
if not self.frameH.is_checked(f"text='{checkbox_label}'"):
    self.frameH.check(f"text='{checkbox_label}'")
    self.save_settings(framehandle=self.frameH)
else:
    print("Category is already checked... ignoring request")
```
DomTree is as follows
```
<span id="thePage:frm:j_id237:URLSettings:j_id253:2:j_id255" class="fs-toggle-checkbox">
   <input id="thePage:frm:j_id237:URLSettings:j_id253:2:category" type="checkbox" name="thePage:frm:j_id237:URLSettings:j_id253:2:category">
   <label for="thePage:frm:j_id237:URLSettings:j_id253:2:category" class="slds-checkbox--button__label">
     <span class="fs-icon-check" id="adultCategory">
           <img src="/resource/1614935913000/afscb__SLDS0203/assets/icons/utility/check_60.png" class="slds-icon slds-icon--selected slds-icon--x-small slds-icon-text-default slds-m-right--x-small">
    </span> adult
  </label>
</span>
```
after upgrading to 1.9 Playwright seems to find element but waits for it to be visible, as can be seen from debug logs in image below
![image](https://user-images.githubusercontent.com/3527464/110582204-85729300-8174-11eb-94af-6c38ec935c37.png)

I then tried to make more precise locator which would pinpoint to the input element as follows --
```
self.frameH.check(f"//label[contains(.,'{checkbox_label}')]/preceding-sibling::input")
```
But that does not help. What helps infact is doing this (_notice its same pinpointed loctor_)
```
self.frameH.eval_on_selector(f"//label[contains(.,'{checkbox_label}')]/preceding-sibling::input", "checkbox =>checkbox.checked=true")
```
I believe there is something broken with 1.9.0 which is the version I am using at the moment.

**Comment by yury-s** [comment]
Sounds like the same issue as https://github.com/microsoft/playwright/issues/5634 which is fixed in v1.9.2. Please reopen if the problem still reproduces with playwright-python 1.9.**2**

**Comment by mrudulp** [comment]
validated this is now fixed with 1.9.2
